decorator

```diff
@@ -0,0 +1,9 @@
+class Decorator < Nameable
```

Comment:
- [ ] Kindly add the extension `.rb` to this file in order to become a ruby file.